workload/src/Samsung.Tizen.Sdk/targets/Samsung.Tizen.Sdk.Versions.targets

```diff
@@ -45,6 +45,21 @@ Copyright (c) Samsung All rights reserved.
   </ItemGroup>
 
   <ItemGroup>
+
+    <KnownFrameworkReference
+      Include="Samsung.Tizen"
+      TargetFramework="net7.0-tizen$(_DefaultTargetPlatformVersion)"
+      TargetingPackName="Samsung.Tizen.Ref"
+      TargetingPackVersion="**FromWorkload**"
+      Profile="Tizen"
+    />
+    <KnownFrameworkReference
+      Include="Samsung.Tizen"
+      TargetFramework="net7.0-tizen$(_DefaultTargetPlatformVersion)"
+      TargetingPackName="Samsung.Tizen.Ref"
+      TargetingPackVersion="**FromWorkload**"
+      Profile="Tizen"
+    />
```

Comment:
Duplicated element.

Reply:
yes, I will remove and update `$(_DefaultTargetPlatformVersion)` to `7.0` (hard coded value)